Reconstruction/RecCalorimeter/options/runEcalReconstruction.py

```diff
@@ -0,0 +1,60 @@
+EVTMAX=100
+
+from Gaudi.Configuration import *
+
+from Configurables import ApplicationMgr, FCCDataSvc, PodioOutput
+
+podioevent   = FCCDataSvc("EventDataSvc")
+
+# reads HepMC text file and write the HepMC::GenEvent to the data service
+from Configurables import PodioInput
+podioinput = PodioInput("PodioReader", filename="/afs/cern.ch/user/n/novaj/public/FCCSW_tests/output_ecal.root", collections=["ECalHits", "ECalClusters", "GenParticles", "GenVertices"], OutputLevel=DEBUG)
```

Comment:
For the time being we put the file here:
`/afs/cern.ch/exp/fcc/sw/0.7/testsamples/output_ecalSim_e50GeV_eta0_10events.root`

Will investigate the EOS seg-fault.